src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java

```diff
@@ -547,8 +546,13 @@ public List<org.candlepin.subscriptions.db.model.Subscription> findSubscriptions
             .billingAccountId(usageKey.getBillingAccountId())
             .payg(true)
             .beginning(rangeStart)
-            .ending(rangeEnd)
-            .build();
+            .ending(rangeEnd);
+
+    ReportCriteria subscriptionCriteria =
+        orgId
+            .map(s -> reportCriteriaBuilder.orgId(s).build())
```

Comment:
Super minor gripe, I think we should stick to using more informational variable names in lambdas. Could you rename 's' to 'id' or something similar?

Reply:
Good suggestion. I have addressed it.